Makefile.cflags

```diff
@@ -66,3 +66,21 @@ endif
 ifeq ($(origin ARFLAGS),default)
   ARFLAGS = rcs
 endif
+
+# check if we want to make a build without any optimalization
+ifeq (1,$(ALL_DEBUG_BUILD))
+  # add the all-debug target to compile witouth optimalization
```

Comment:
That line can be removed btw... its a unmodified copy from above. Can fix
tonight.
On 6 Apr 2016 07:53, "Joakim Nohlgård" <notifications@github.com> wrote:

> In Makefile.cflags
> <https://github.com/RIOT-OS/RIOT/pull/4413#discussion_r58654038>:
>
> > @@ -66,3 +66,21 @@ endif
> >  ifeq ($(origin ARFLAGS),default)
> >    ARFLAGS = rcs
> >  endif
> > +
> > +# check if we want to make a build without any optimalization
> > +ifeq (1,$(ALL_DEBUG_BUILD))
> > +  # add the all-debug target to compile witouth optimalization
>
> Sorry, I found one more: "optimalization" -> "optimization"
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/4413/files/93af64e07dd03e13ad8bcb8c21afc73b3fdf9425#r58654038>
>